### DISCUSSION TITLE
Grayed out non-clickable Button

### DISCUSSION THREAD

**Discussion by destruct1** [zauberzeug/nicegui#480] [Q&A] [answered]
Hello!

Is it possible to gray out the button element so that it can't be clicked, and have it be able to have the "clickable" state to be toggled?

Thank you!

**Comment by rodja** [comment, marked as answer]
Like this?

```py
b = ui.button('button', on_click=lambda: ui.notify('clicked'))
ui.switch(value=True, on_change=lambda e: b.props(remove='disabled') if e.value else b.props('disabled'))
```

**Reply by falkoschindler** [reply]
Note that - as noticed in #560 - it might be better to use the Quasar prop `disable` (without "d") instead of the HTML attribute `disabled`.
